# This expresses the history of a robot
int16 id
float32[] specs
int16[] e_v
int16[] e_uv
int16[] e_g
Intlist[] T_a
int16[] T_f
int16   currEdge
int16   nextNode
float32[]   pose
int16[] lastMeeting
int16[] Whole_path
bool abort_curr_edge
bool available
